testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -244,6 +244,21 @@
   end
 end
 
+#
+# Click on a button with a given "id"
+When(/^I click on button with id "([^"]*)"$/) do |button_id|
+  find(:xpath, ".//button[@id=\"#{button_id}\"]").click
+end
+
```

Comment:
You don't need a special step to click by Id.

The step `I click on "zzz"` will check in both text and ids.

![image](https://user-images.githubusercontent.com/2827771/116702388-4ff66300-a9c9-11eb-83ca-bc52fd2fcd70.png)

Reply:
In any case, as I commented in another part of the PR, I strongly recommend to search by text and not by ID.
As it's easier to follow the steps by someone that reads the .feaure file.

Reply:
Ahhh! I can pass the `id` directly to the "I click on" step.

Ok. I'm fixing this - Thanks!